test/functional/shell/viml_system_spec.lua

```diff
@@ -184,14 +184,16 @@ describe('system()', function()
     end)
   end)
 
-  if xclip then
     describe("with a program that doesn't close stdout", function()
-      it('will exit properly after passing input', function()
-        eq('', eval([[system('xclip -i -selection clipboard', 'clip-data')]]))
-        eq('clip-data', eval([[system('xclip -o -selection clipboard')]]))
-      end)
+      if not xclip then
+        pending('skipped (missing xclip)')
+      else
+        it('will exit properly after passing input', function()
+          eq('', eval([[system('xclip -i -selection clipboard', 'clip-data')]]))
+          eq('clip-data', eval([[system('xclip -o -selection clipboard')]]))
+        end)
+      end
     end)
-  end
```

Comment:
The indentation is off for this paragraph.
Assuming you saw my deleted comments, then sorry for the confusion